pkg/sql/colexec/execgen/inline.go

```diff
@@ -0,0 +1,363 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package execgen
+
+import (
+	"bytes"
+	"fmt"
+	"go/parser"
+	"go/token"
+
+	"github.com/cockroachdb/cockroach/pkg/testutils/lint/passes/fmtsafe/testdata/src/github.com/cockroachdb/errors"
```

Comment:
Is this really the package import -- this is like 150 characters long

Reply:
lol, wtf. fixed.